2_drivers.yml

```diff
@@ -17,13 +17,39 @@ targets:
   #out_data/placeholder_for_gefs_archive:
    # command: copy_files()
     
-  #out_data/palceholder_for_nldas:
-   # command: copy_files()
-   
+  # NLDAS Files
+  ## NLDAS_time[0.377900]_x[397]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[397]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[397]_y[136].csv',
+    to = target_name)
+  ## NLDAS_time[0.377900]_x[401]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[401]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[401]_y[136].csv',
+    to = target_name)
+  
+  ## NLDAS to reservoir meteo rds  
+  log/reservoir_meteo_files.yml:
+    command: fetch_meteo_files(
+      out_yml = target_name,
+      nml_rds = 'out_data/reservoir_nml_list.rds')
+  
+  # GEFS archive
+  out_data/met_drivers.zip:
+    command: utils::zip(files = 'in_data/met_drivers', zipfile = target_name)
+  
+  # Observed flow and temp at ins/outs of reservoirs for glm 
   out_data/reservoir_io_obs.csv:
     command: copy_filter_feather(
       out_csv = target_name,
       in_feather = 'in_data/res_io_obs.feather',
       site_ids = reservoir_modeling_site_ids)
+  
+  # modeled flow and temperature at ins/outs of reservoirs for GLM
+  out_data/reservoir_io_sntemp.csv:
```

Comment:
Does this file, and the res_id_obs.feather file need any preceding transformation like what occurs in[ delaware-model-prep](https://github.com/USGS-R/delaware-model-prep/blob/c12ca802415e2689bf71e7b25055874fa1165e39/9_collaborator_data.yml#L315).  Or is that already done when it was copied over to the data_release in folder.